src/FilterRule/NumberFormat.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Particle.
+ *
+ * @link      http://github.com/particle-php for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Particle (http://particle-php.com)
+ * @license   https://github.com/particle-php/Filter/blob/master/LICENSE New BSD License
+ */
+namespace Particle\Filter\FilterRule;
+
+use Particle\Filter\FilterRule;
+
+/**
+ * Class NumberFormat
+ *
+ * @package Particle\Filter\FilterRule
+ */
+class NumberFormat extends FilterRule
+{
+    /**
+     * @var int
+     */
+    protected $decimals;
```

Comment:
We haven't been very strict on this before, and personally I don't feel the need. I see how it would be the more logical thing to do though. Unless there are bigger, underlying issues with the protected declaration, which I'm unaware off (speed maybe?).